h/static/images/icons/info_icon.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="12px" height="12px" viewBox="0 0 12 12" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
I see you've removed the `<title>` and `<desc>`. That's fine, though not strictly required. The `svg_icon()` helper will actually strip out the `title` tag itself. The idea is that we can add SVGs generated by tools like Sketch to the source tree without any manual edits to the file.